components/identities/actions/identityActions.tsx

```
@@ -319,21 +318,72 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
-                    </p>
-                  )}
+                    ) : (
+                      <p
+                        onClick={() => setViewMoreClicked(true)}
+                        className={styles.viewMore}
+                      >
+                        View more
+                      </p>
+                    )}
+
+                  </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)